sdk/communication/communication-rooms/README.md

```diff
@@ -124,10 +127,18 @@ To update the `validFrom` and `validUntil` settings of a room use the `updateRoo
 
 Starting in 1.1.0 release, `PstnDialOutEnabled` property is added to enable or disable PSTN Dial-Out feature in a room.
 
-```typescript
-validForDays = 60;
+```ts snippet:ReadmeSampleUpdateRoom
+import { DefaultAzureCredential } from "@azure/identity";
+import { RoomsClient, UpdateRoomOptions } from "@azure/communication-rooms";
+
+const credential = new DefaultAzureCredential();
+const client = new RoomsClient("<endpoint-from-resource>", credential);
+
+const validFrom = new Date(Date.now());
+const validForDays = 60;
+const validUntil = new Date(validFrom.getTime());
 validUntil.setDate(validFrom.getDate() + validForDays);
```

Comment:
Isn't this wrong? `getDate()` return the day in the month 1...31, and +60 puts it out of bounds?

Reply:
No, it updates the date appropriately.  See the [documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setDate#description)

Reply:
Sometimes I forget that JavaScript is just special.